tests/unittests/CoreGraphics/CGPathTests.mm

```diff
@@ -981,4 +981,52 @@ static bool testSymmetricEquivalence(CGPathRef path1, CGPathRef path2) {
     EXPECT_EQ(boundingBox.origin, CGPointMake(50, 50));
     EXPECT_EQ(boundingBox.size, CGSizeMake(50, 50));
     CGPathRelease(thepath);
+}
+
+static void inline CGRectCompareWithSlack(CGRect lhs, CGRect rhs) {
+    EXPECT_NEAR(lhs.origin.x, rhs.origin.x, .001);
```

Comment:
This should be a macro. otherwise, every test failure will point to this function instead of the actual place the rect is being compared.